### PR TITLE
Fix source container deadline

### PR DESCRIPTION
**Pull request by MartinBasti**
# Maintainers will complete the following section

- [x] Commit messages are descriptive enough
- [ ] Code coverage from testing does not decrease and new code is covered
- [n/a] JSON/YAML configuration changes are updated in the relevant schema
- [n/a] Changes to metadata also update the documentation for the metadata
- [n/a] Pull request has a link to an osbs-docs PR for user documentation updates

### CODE REVIEW

File: tests/build_/test_build_requestv2.py [hunk 1]
```diff
@@ -1266,3 +1266,31 @@ def test_render_scratch_source_request(self):
...
+           'worker_max_run_hours': worker_max,
+           'orchestrator_max_run_hours': orchestrator_max,
+           'build_from': 'image:fedora:latest',
+        }
+        user_params = get_sample_source_params(conf_args=conf_args)
+        build_request = SourceBuildRequest(osbs_api=MockOSBSApi(), user_params=user_params)
+
+        build_json = build_request.render()
+        if expected:
+            expected_hours = expected * 3600
```
Comment (lkolacek) [inline tests/build_/test_build_requestv2.py:1292]: This value will be probably expected_seconds

Comment (MartinBasti) [reply tests/build_/test_build_requestv2.py]: Fixed in additional commit (I also fixed it in another testcase)

**Review by ssalatsk — APPROVED** [review]

**Comment by rcerven** [comment]
code looks good, tests failing

**Comment by MartinBasti** [comment]
Added commit that fixes coveralls

**Review by rcerven — APPROVED** [review]

**Review by lkolacek — APPROVED** [review]

**Comment by MartinBasti** [comment]
release notes:
- Source container image builds have now set completion deadline

**Pull request merged**